package.json

```diff
@@ -117,12 +119,13 @@
     "karma-jasmine": "~4.0.1",
     "karma-jasmine-html-reporter": "^1.7.0",
     "ng-packagr": "^13.2.1",
+    "ngx-build-plus": "^13.0.1",
     "prettier": "^2.5.1",
-    "protractor": "~7.0.0",
     "puppeteer": "^13.4.0",
     "sass": "^1.49.9",
     "ts-node": "^10.5.0",
-    "typescript": "4.5.5"
+    "typescript": "4.5.5",
+    "cypress": "latest"
```

Comment:
I can see that the draft PR is merged already. I kind of thought that you will change this line before merge. Is there some good reason why to allow uncontrollably install any latest version of cypress @DailisLangovskis ?

Reply:
@jmacura Well if You point it out then no, it was added after installing, I just forgot to fix it as well :)